src/Console/ConfigurationResolver.php

```diff
@@ -493,6 +498,14 @@ public function shouldStopOnViolation()
     }
 
     /**
+     * @return null|bool
+     */
+    public function configFinderIsOverridden()
+    {
+        return $this->configFinderOverridden;
```

Comment:
this shall never return null. 
all properties here are nullable, but getters are not providing nulls. null means here only "lazy computed"